### PR TITLE
Change Advanced Configurations fields of Proxy

### PR DESCRIPTION
**Pull request by Bhashinee**
Removed the "Advanced Configurations" field when Custom Proxy is selected, since it does not have anything configurable.
Related Issue : https://github.com/wso2/devstudio-tooling-ei/issues/48

### CODE REVIEW

File: plugins/org.wso2.developerstudio.eclipse.artifact.proxyservice/src/org/wso2/developerstudio/eclipse/artifact/proxyservice/ui/wizard/AdvancedConfigComposite.java [hunk 1]
```diff
@@ -175,7 +192,7 @@ public void widgetSelected(SelectionEvent e) {
 		};
 
 		optEnterURL.setText("Enter URL");
-
+
```
Comment (vinok88) [inline plugins/org.wso2.developerstudio.eclipse.artifact.proxyservice/src/org/wso2/developerstudio/eclipse/artifact/proxyservice/ui/wizard/AdvancedConfigComposite.java]: Unnecessary space

File: plugins/org.wso2.developerstudio.eclipse.artifact.proxyservice/src/org/wso2/developerstudio/eclipse/artifact/proxyservice/ui/wizard/AdvancedConfigComposite.java [hunk 2]
```diff
@@ -137,6 +142,18 @@ public AdvancedConfigComposite(Composite parent, int style,
 		setModel((ProxyServiceModel) getProjectModel());
 
 		/* common */
+		
+		txtAdvancedConfig = new StyledText(this, SWT.NONE);
+		txtAdvancedConfig.setLayoutData(new GridData(SWT.FILL, SWT.CENTER,
```
Comment (nuwanw) [inline plugins/org.wso2.developerstudio.eclipse.artifact.proxyservice/src/org/wso2/developerstudio/eclipse/artifact/proxyservice/ui/wizard/AdvancedConfigComposite.java]: use wso2 code formatting style

Comment (Bhashinee) [inline plugins/org.wso2.developerstudio.eclipse.artifact.proxyservice/src/org/wso2/developerstudio/eclipse/artifact/proxyservice/ui/wizard/AdvancedConfigComposite.java]: Fixed.

File: plugins/org.wso2.developerstudio.eclipse.artifact.proxyservice/src/org/wso2/developerstudio/eclipse/artifact/proxyservice/ui/wizard/AdvancedConfigComposite.java (hunk 1 above)
Comment (Bhashinee) [inline plugins/org.wso2.developerstudio.eclipse.artifact.proxyservice/src/org/wso2/developerstudio/eclipse/artifact/proxyservice/ui/wizard/AdvancedConfigComposite.java]: Removed it.

File: plugins/org.wso2.developerstudio.eclipse.artifact.proxyservice/project_wizard.xml [hunk 3]
```diff
@@ -18,8 +18,7 @@
 			<data modelProperty="create.esb.prj" type="link" fieldController="org.wso2.developerstudio.eclipse.artifact.proxyservice.validators.ProxyServiceProjectFieldController" controlData="align=right">&lt;a&gt;Create new ESB Project...&lt;/a&gt;</data>
  
 			<data group="group1" modelProperty="proxy.AdvancedConfig" type="composite"  fieldController="org.wso2.developerstudio.eclipse.artifact.proxyservice.validators.ProxyServiceProjectFieldController" controlData="class=org.wso2.developerstudio.eclipse.artifact.proxyservice.helper.AdvancedConfigProvider">Configuration</data>
-			<group id="group1" controlData="type=collapsible;state=expanded"></group>
-			
+			<group id="group1" controlData="type=collapsible;state=expanded"></group>			
 		</settings>
```
Comment (sohaniwso2) [inline plugins/org.wso2.developerstudio.eclipse.artifact.proxyservice/project_wizard.xml]: unnecessary line

Comment (sohaniwso2) [inline plugins/org.wso2.developerstudio.eclipse.artifact.proxyservice/project_wizard.xml]: unnecessary line

File: plugins/org.wso2.developerstudio.eclipse.artifact.proxyservice/src/org/wso2/developerstudio/eclipse/artifact/proxyservice/ui/wizard/AdvancedConfigComposite.java [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
  *
```
Comment (sohaniwso2) [inline plugins/org.wso2.developerstudio.eclipse.artifact.proxyservice/src/org/wso2/developerstudio/eclipse/artifact/proxyservice/ui/wizard/AdvancedConfigComposite.java]: this should be 2012-2017

**Pull request merged**